pkg/cli/demo_cluster.go

```diff
@@ -907,3 +907,10 @@ func (c *transientCluster) listDemoNodes(w io.Writer, justOne bool) {
 		fmt.Fprintln(w, `To display connection parameters for other nodes, use \demo ls.`)
 	}
 }
+
+// genRandomDemoPassword generates a password that prevents accidental
+// misuse of the DB console started by demo shells.
+func genRandomDemoPassword(username string) string {
+	mypid := os.Getpid()
+	return fmt.Sprintf("%s%d", username, mypid)
```

Comment:
non-blocking q: do we still want `pid`? i imagine if we were super lazy (esp for demo mode), `demo` would be fine :P